src/shared-styles/_typography.scss

```diff
@@ -35,6 +37,24 @@
   @include _typography($scale-point, $baseline-units, font.$secondary, 600);
 }
 
+@mixin _label-typography($color, $uppercase: true) {
```

Comment:
It's generally considered more readable to have private members below public ones, so  move this below `@mixin _typography`